src/getter.js

```diff
@@ -9,6 +11,19 @@ import { isKeyPath } from './key-path'
  */
 const identity = (x) => x
 
+class GetterClass {
```

Comment:
yes, but called `Getter` instead of `addOptionsToGetter`.  The latter name implied that you could mutate an existing Getter (potentially globally) versus a function to create a Getter.